update_cmake_build.bat

```diff
@@ -31,12 +31,7 @@ copy ".\readme.*"  %mmex_build_dir%
 @echo Updating Help files for: %current_build_type%
 @echo ------------------------------------------------------------------------
 if not exist %mmex_build_dir%\help mkdir %mmex_build_dir%\help
-copy ".\doc\help\*.*"                   "%mmex_build_dir%\help"
-@echo.
-for /f %%d IN ('dir /A:D /B .\doc\help') do (
-    if not exist "%mmex_build_dir%\help\%%d" mkdir "%mmex_build_dir%\help\%%d"
-    copy ".\doc\help\%%d\*.*"  "%mmex_build_dir%\help\%%d"
-)
+xcopy /Y /I /S ".\docs\*.*"                   "%mmex_build_dir%\help"
```

Comment:
According to my [proposal](https://github.com/moneymanagerex/moneymanagerex/pull/1136/files#diff-af3b638bc2a3e6c650974192a53c7291R271) to rename dir `docs` to `help`, this line needs to be altered too.